lightningd/peer_htlcs.c

```diff
@@ -478,28 +478,35 @@ static void forward_htlc(struct htlc_in *hin,
 			 const u8 next_onion[TOTAL_PACKET_SIZE])
 {
 	enum onion_type failcode;
-	u64 fee;
 	struct lightningd *ld = hin->key.channel->peer->ld;
 	struct channel *next = active_channel_by_id(ld, next_hop, NULL);
 
+	u64 fee;
+	u32 fee_base = ld->config.fee_base;
+
 	/* Unknown peer, or peer not ready. */
 	if (!next || !next->scid) {
 		local_fail_htlc(hin, WIRE_UNKNOWN_NEXT_PEER, NULL);
 		return;
 	}
 
+	/* Fee base and ppm can also be set per channel */
+	if (next->feerate_base > 0) {
+		fee_base = next->feerate_base;
+	}
+	// TODO: next->feerate_ppm
```

Comment:
The commit in which you address this should be squashed with this one, to keep changes logically close together.